src/makeInfinityAdder.js

```diff
@@ -3,8 +3,24 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(num) {
+  let sum = num || 0;
+
+  function result(nextNum) {
```

Comment:
The parameter `nextNum` should not have a default value set. The task specifies that there is no need to set a default value for the `adder` function argument.